pkg/cmd/extension/command.go

```diff
@@ -293,19 +293,35 @@ func NewCmdExtension(f *cmdutil.Factory) *cobra.Command {
 				Use:   "install <repository>",
 				Short: "Install a gh extension from a repository",
 				Long: heredoc.Docf(`
-					Install a GitHub repository locally as a GitHub CLI extension.
+					Install a GitHub CLI extension from a GitHub repository.
 
-					The repository argument can be specified in %[1]sOWNER/REPO%[1]s format as well as a full URL.
+					The repository argument can be specified in %[1]sOWNER/REPO%[1]s format or as a full repository URL.
 					The URL format is useful when the repository is not hosted on github.com.
 
-					To install an extension in development from the current directory, use %[1]s.%[1]s as the
-					value of the repository argument.
+					To install an in-development extension from a locally cloned repository, use %[1]s.%[1]s as the
+					value of the repository argument. Note the following:
```

Comment:
Continued from above
```suggestion
					For local repositories, often used while developing extensions, use %[1]s.%[1]s as the
					value of the repository argument. Note the following:
```